src/applications/facility-locator/api/index.js

```diff
@@ -3,4 +3,4 @@ import MockApi from './MockLocatorApi';
 import LiveApi from './LocatorApi';
 
 /* global __BUILDTYPE__ */
-export default (__BUILDTYPE__ === 'development') ? MockApi : LiveApi;
+export default (__BUILDTYPE__ === 'localhost') ? MockApi : LiveApi;
```

Comment:
They might be intending to run their mock api in dev.va.gov. I vaguely remember that being the case.

Reply:
Left a comment over here, https://github.com/department-of-veterans-affairs/vets-website/pull/8651#issuecomment-438314848.

Reply:
My [feedback](https://github.com/department-of-veterans-affairs/vets-website/pull/8651#issuecomment-438377371) on this.